standings.js

```diff
@@ -61,13 +61,24 @@ fetchJSON("standings.json", function (gameResults) {
       return byWinningPercentage
     })
 
+const gamesBehindByTeam = sortedStandings.map((team) => {
+  const gamesBehind = ((sortedStandings[0].w - team.w) - (sortedStandings[0].l - team.l)) / 2
+  team.gb = gamesBehind
```

Comment:
I don't like this as a means of setting each team's GB stat. It's added on as a side-effect of the rest of the work, which is confusing and unexpected.